content/api/account.apib

```diff
@@ -37,6 +37,8 @@ Retrieve information regarding your SparkPost account and set account options.
         + rest_tracking_default (boolean) - Account-level default for REST API engagement tracking. `true` by default.
         + transactional_unsub (boolean) - Set to `true` to include `List-Unsubscribe` header for all transactional messages by default
         + transactional_default (boolean) - Set to `true` to send messages as transactional by default
+        + initial_open_include_alt_attribute (boolean)
+            Account-level initial open html `alt` tag. `false` by default. To include an alt attribute (alt="") in initial open pixel tracking link tags set the `initial_open_include_alt_attribute` to `true`. Initial open pixel tracking must be enabled.
```

Comment:
This should read "Account-level option for adding html `alt` tag to initial open".  The first sentence doesn't really make sense.